src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -734,15 +1026,22 @@ public Task GoToAsync(ShellNavigationState state, bool animate, ShellNavigationQ
 		/// <summary>Bindable property for <see cref="CurrentState"/>.</summary>
 		public static readonly BindableProperty CurrentStateProperty = CurrentStatePropertyKey.BindableProperty;
 
-		/// <summary>Bindable property for <see cref="FlyoutBackgroundImage"/>.</summary>
+		/// <summary>
+		/// Sets the flyout background image, of type ImageSource, to a file, embedded resource, URI, or stream.
```

Comment:
I would add the `of type ImageSource` part to the remarks section, along with the second sentence too.

Reply:
Added.